tasks/check-for-sessions.yml

```diff
@@ -0,0 +1,19 @@
+---
+  - name: Check for running meetings
+    uri:
+      url: "https://{{ bbb_hostname }}/bigbluebutton/api/getMeetings?checksum={{ ('getMeetings' + bbb_secret) | hash('sha1') }}"
+      return_content: true
+    register: bbb_getmeetings
+    ignore_errors: true
+    tags: always
```

Comment:
I am not happy with this solution.
`ignore_errors: true` is considered a bad practice for a reason.

The current output when running this PR on a new hosts creates this error:
```
TASK [ansible-role-bigbluebutton : Check for running meetings] ******************************************************************************
fatal: [meet.bbb.ansibletest.safeserver-support.de]: FAILED! => {"changed": false, "content": "", "elapsed": 0, "msg": "Status code was -1 and not [200]: Request failed: <urlopen error [Errno 111] Connection refused>", "redirected": false, "status": -1, "url": "https://meet.bbb.ansibletest.safeserver-support.de/bigbluebutton/api/getMeetings?checksum=REDACTED"}
...ignoring
```
At least this is the error I got.
I would prefer a [`failed_when:`](https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html) condition.

Your thoughts?

Reply:
The idea is to never have that task fail because that would stop the ansible run. I refer to the result in the next task and stop the play for a specific host, if needed. Perhaps we can come up with something like `failed_when: never`.

Reply:
I just found https://medium.com/@sbarnea/why-ansible-ignore-errors-is-evil-500fb6e81229 which answered my question "why is it bad practice" and supported my Idea of `failed_when: never` - just by way of:
```
- name: Do not count this as a failure
  ansible.builtin.command: /bin/false
  failed_when: false
  changed_when: false